### ISSUE TITLE
It would be nice if the configuration file for nrpe allowed non-ssl mode.

### ISSUE THREAD

**Issue by dswartz** [astersmith/esos#152]
The nrpe daemon has a '-n' flag that stipulates disabling SSL.  If would be nice to be able to specify that in the nrpe config file (/etc/rc.d/rc.nrpe).  I will open a pull request for this...

**Comment by msmith626** [comment]
I've completed this and testing it now... will have it committed this afternoon.

--Marc

**Comment by dswartz** [comment]
On 2015-10-14 11:24, Marc A. Smith wrote:
> I've completed this and testing it now... will have it committed this
> afternoon.

Cool, thanks.  I may get to test it in the next couple of days.  I am 
replacing my sandy bridge server with 32GB ram (3.2 ghz 4-core xeon) 
with a dell R905 I got on ebay for $335 :)  128GB ram and four 4-core 
2.7ghz AMD cpus.  With any luck, I can test this at the same time :)

**Comment by msmith626** [comment]
This is complete (dde411a59b8444b74ed325bf8a4151e0f13e8fc4); I did some limited testing and it seems to work fine. It even handles in the case of an upgrade where the /etc/rc.conf doesn't contain default values.

Let me know how it goes... I also did added some brief documentation to the wiki for this. See the new /etc/rc.conf (in master branch under etc/) as if you upgrade you won't get these changes.


--Marc
